clang/lib/Driver/Driver.cpp

```diff
@@ -3498,7 +3498,8 @@ class OffloadingActionBuilder final {
       // Do not create an unbundling action for an object when we know a fat
       // static library is being used.  A separate unbundling action is created
       // for all objects and the fat static library.
-      if (!(HostAction->getType() == types::TY_Object &&
+      if (C.getDefaultToolChain().getTriple().isWindowsMSVCEnvironment() ||
```

Comment:
please extend comment why Windows is handled the same way as !(object && static_lib) case here